main/main.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/coreos/clair/pkg/commonerr"
+	"github.com/coreos/clair/pkg/httputil"
+	errs "github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/k8s-istio-cve-pusher/utils"
+)
+
+const (
+	dataFeedURL      = "https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-%s.json.gz"
+	dataFeedMetaURL  = "https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-%s.meta"
+	logDataFeedName  = "data feed name"
+	kubernetes       = "k8s"
+	istio            = "istio"
+	fileExt          = ".json"
+	k8sCveFileName   = "NVDk8sCVEs.json"
+	istioCveFileName = "NVDistioCVEs.json"
+)
+
+type NVDMetadata struct {
+	CVSSv2 NVDmetadataCVSSv2
+	CVSSv3 NVDmetadataCVSSv3
+}
+
+type NVDmetadataCVSSv2 struct {
+	PublishedDateTime string
+	Vectors           string
+	Score             float64
+}
+
+type NVDmetadataCVSSv3 struct {
+	Vectors             string
+	Score               float64
+	ExploitabilityScore float64
+	ImpactScore         float64
+}
+
+func main() {
+	if err := runCmd(); err != nil {
+		log.Fatal(err)
+		os.Exit(1)
+	}
+}
+
+func runCmd() error {
+	path, err := utils.IsEnvVarNonEmpty("LOCAL_FILE_PATH")
+	if err != nil {
+		return err
+	}
+
+	if err = utils.IsPathWritableDir(path); err != nil {
+		return err
+	}
+
+	if err = utils.IsgcloudConfigOK(); err != nil {
+		return err
+	}
+
+	feedReader, err := getDataFeeds(path)
+	if err != nil {
+		return errs.Wrap(err, "error downloading from NVD DB")
+	}
+
+	k8sCVEs, istioCVEs, err := getK8sAndIstioCVEs(feedReader)
+	if err != nil {
+		return err
+	}
+
+	k8sFileFullName := filepath.Join(path, k8sCveFileName)
+	if err = utils.StoreCVEsToFile(k8sCVEs, k8sFileFullName); err != nil {
+		return err
+	}
+
+	istioFileFullName := filepath.Join(path, istioCveFileName)
+	if err = utils.StoreCVEsToFile(istioCVEs, istioFileFullName); err != nil {
+		return err
+	}
+
+	bucket := os.Getenv(utils.GcloudBucket)
+	if err = utils.WriteToBucket(bucket, filepath.Join(kubernetes, k8sCveFileName), k8sFileFullName); err != nil {
+		return err
+	}
+	log.Infof("done pushing to bucket path %q", filepath.Join(bucket, kubernetes, k8sCveFileName))
+
+	if err = utils.WriteToBucket(bucket, filepath.Join(istio, istioCveFileName), istioFileFullName); err != nil {
+		return err
+	}
+	log.Infof("done pushing to bucket path %q", filepath.Join(bucket, istio, istioCveFileName))
+	return nil
+}
+
+func getK8sAndIstioCVEs(feedReader map[string]string) ([]utils.NvdCVEEntry, []utils.NvdCVEEntry, error) {
+	var allk8sCVEs, allistioCVEs []utils.NvdCVEEntry
```

Comment:
Capitalize the beginning of every word, regardless of whether you'd spell it in lowercase usually, i.e., `allK8s...`